### PR TITLE
[dependabot][mergify] automate merge and add reviewers

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

- Add reviewers
- Automerge if CI is happy. Although, if no approval is needed why do we need the reviewers?

## Why is it important?

Look mum no hands!

## Issue

See https://github.com/elastic/integrations/pull/1291#issuecomment-876963997

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -6,5 +6,7 @@ updates:
     schedule:
       interval: "daily"
     labels:
-      - dependencies
-      - go
+      - automation
+    reviewers:
+      - "mtojek"
+      - "jsoriano"
```
Comment (jsoriano) [inline .github/dependabot.yml:12]: I'd prefer to put a [github] team here so we can more easily manage notifications and we don't need need to change this if the people related to the project changes. But we can go on like this by now.

**Review by jsoriano — APPROVED** [review]

**Review by mtojek — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (v1v) [reply .github/dependabot.yml:12]:
```suggestion
      - "elastic/integrations-developer-experience"
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -6,5 +6,7 @@ updates:
     schedule:
       interval: "daily"
     labels:
-      - dependencies
-      - go
+      - automation
+    reviewers:
+      - "mtojek"
```
Comment (v1v) [inline .github/dependabot.yml:11]:
```suggestion
      - "elastic/integrations-developer-experience"
```

**Review by mtojek — APPROVED** [review]
> Automerge if CI is happy. Although, if no approval is needed why do we need the reviewers?

Personally I use it for email notifications :)

**Review by jsoriano — APPROVED** [review]

**Pull request merged**